ci/steps.yml

```diff
@@ -67,7 +92,7 @@ steps:
       export ANDROID_NDK_ROOT=$(Build.SourcesDirectory)/android-ndk-r21e
       mkdir $(Build.BinariesDirectory)/tests
       (cd $(Build.BinariesDirectory)/tests; 7zr x $(Build.SourcesDirectory)/ci/accelbubble.7z)
-      export PATH=$(Build.BinariesDirectory)/Qt/$(QT_VERSION)/$(ARCHDIR)/bin:$PATH
+      export PATH=$(QT_BINDIR):$PATH
```

Comment:
This is part of a global replacement of `$(Build.BinariesDirectory)/Qt/$(QT_VERSION)/$(ARCHDIR)/bin` for `$(QT_BINDIR). I found it simpler to use a variable than to conditionally change the PATH variable.